langchain/evaluation/run_evaluators/implementations.py

```diff
@@ -111,10 +112,12 @@ def get_qa_evaluator(
             choices_map={"CORRECT": 1, "INCORRECT": 0},
         ),
     )
+    tags = kwargs.pop("tags", [])
     return RunEvaluatorChain(
         eval_chain=eval_chain,
         input_mapper=input_mapper,
         output_parser=output_parser,
+        tags=tags + [evaluation_name],
```

Comment:
This makes it nicer in the UI since the feedback runs are logged in your debug session (or whatever is configured outside)